LinkedList.cs

```diff
@@ -19,15 +19,42 @@ public Node(T nodeInput)
          class  LinkedList<T>
         {
             public   Node<T> Head;
-            public   int Length = 0;
+            public int Length = 0;
 
-            public  void Prepend( T prependValue)
+            public void Prepend(T prependValue)
             {
                 Node<T> PrependThis = new Node<T>(prependValue);
                 PrependThis.next = Head;
                 Head = PrependThis;
                 Length++;
             }
+            
+            public void Append(T appendValue)
+            {
+                var tempValue = Head;
+                var appendHere = tempValue;
+                Node<T> AppendThis = new Node<T>(appendValue);
+                if(Length == 0)
+                {
+                    AppendThis.next = Head;
+                    Head = AppendThis;
```

Comment:
When Length is 0,
Head will be null, no need to set it

Reply:
So net nodig om: Head = AppendThis ?